docs/topics/proc_reporting-issues.adoc

```diff
@@ -15,7 +15,7 @@ You must sign up for a Jira account in order to create an issue.
 
 .Procedure
 
-. Open a browser and navigate to the Jira link:https://issues.jboss.org/secure/CreateIssue!default.jspa[Create Issue] page.
+. Open a browser and navigate to the Jira link:https://issues.redhat.com/projects/WINDUP[Create Issue] page.
```

Comment:
Honestly, I think this is a clearcut case for creating an attribute with the Jira link in common-attributes.

Also, the link text is misleading. It is not a "Create Issue" page. It is the "Windup project". You may want to create an attribute with the full link, instead of just the URL.

Reply:
Also, I would not include this procedure. You are documenting Windup, not Jira. Just tell them to create a Jira issue, like it says in the other modules.

Reply:
Should I change the instruction to "Open a browser, navigate to the Jira [Windup project], and create an issue" or is the Create button prominent enough that our users will know what to do?